src/Provider.js

```diff
@@ -34,6 +35,8 @@ const DEFAULT_GRANT_TYPES_SUPPORTED = [
 ]
 const DEFAULT_SUBJECT_TYPES_SUPPORTED = ['public']
 
+const DEFAULT_TOKEN_TYPES_SUPPORTED = ['legacyPop', 'dpop']
```

Comment:
Should the constants defined in `src/handlers/AuthenticationRequest.js` be used here ?